src/tests/functional/plugin/conformance/subgraphs_dumper/CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ if(TARGET openvino_paddle_frontend)
     list(APPEND DEFINITIONS ENABLE_OV_PADDLE_FRONTEND)
 endif()
 
+if(TARGET openvino_pytorch_frontend)
+    list(APPEND DEPENDENCIES openvino_pytorch_frontend)
+    list(APPEND DEFINITIONS ENABLE_OV_PYTORCH_FRONTEND)
+endif()
+
```

Comment:
why is it changed? Please clarify

Reply:
Just adding pytorch define into tests, not required right now, but I think it is not a problem to have it here.